lib/contracts/builtin_contracts.rb

```diff
@@ -122,7 +124,7 @@ def valid?(val)
     end
 
     def to_s
-      @vals[0, @vals.size-1].join(", ") + " or " + @vals[-1].to_s
+      @vals[0, @vals.size-1].map { |x| InspectWrapper.new(x) }.join(", ") + " or " + InspectWrapper.new(@vals[-1]).to_s
```

Comment:
Nice idea. It's turning out to be a handy class, I may use it in YARD output too.